src/AsyncAPI.Saunter.Generator.Cli/ToFile/AsyncApiDocumentExtractor.cs

```diff
@@ -0,0 +1,72 @@
+﻿using LEGO.AsyncAPI.Models;
```

Comment:
The new repo for this is under bytebard rather than LEGO. 
Same codebase (direct fork) but actively maintained again (as I'm not at LEGO anymore, and they have dropped maintaining my baby)

Preview has v3 support, while main has a few bug fixes not in the LEGO repo.